components/cta/api.js

```diff
@@ -12,12 +11,9 @@ export function submitFormData(
   axios
     .post(config.API_BASE + "/api/send-contact-mail", formData)
     .then(() => {
-      setShowSuccessMessage(true);
+      window.location.href = config.CANOPAS_URL + "/thank-you";
+      localStorage.setItem("client-name", JSON.stringify(formData.name));
```

Comment:
Let's first store key then redirect

Reply:
i have change this commit but it not display here. i don't know but it push that change in this branch see [here](https://github.com/canopas/canopas-blog/blob/fix-cta-section-issues/components/cta/api.js#L15)